src/bin/julialauncher.rs

```diff
@@ -132,6 +134,119 @@ fn run_selfupdate(_config_file: &juliaup::config_file::JuliaupReadonlyConfigFile
     Ok(())
 }
 
+fn is_interactive() -> bool {
+    std::io::stdin().is_terminal() && std::io::stderr().is_terminal()
```

Comment:
I think this is a necessary condition, but I think in addition we need to parse the command line arguments and figure out whether the Julia process will jump into the REPL or execute either a script/code snippet. I think generally that if someone runs `julia +FOO main.jl`we should not show anything interactive, regardless of whether things are running in a terminal or not.

I think the PR I linked to above should have code that does that.

Reply:
I couldn't find that check in the linked PR, but added this check based on julia args.